build.gradle

```diff
@@ -102,3 +107,9 @@ ext {
     targetSdkVersion = 30
     compileSdkVersion = 30
 }
+
+nexusStaging {
+    username = findProperty("NEXUS_USERNAME")
+    password = findProperty("NEXUS_PASSWORD")
+    stagingProfileId = "ea09119de9f4"
```

Comment:
Maybe it would be better if we put this value into env variable as well?

Reply:
We can do that 👍 However the `stagingProfileId` is not a secret (I follow the same approach we use for detekt: https://github.com/detekt/detekt/blob/7f2a4787a9d78ce32b6f2b4a4df2bd2d571ed5ac/buildSrc/src/main/kotlin/packaging.gradle.kts#L17)